### PR TITLE
Skipping more modules on sonar analysis

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Skips more modules on Sonar Analysis. For some reason there are JDK failures on these modules. I'm also open to more elegant solutions but this is currently failing on the build. This change passed on this build: https://jenkins.phx.connexta.com/service/jenkins/job/Alliance/job/master/809/

#### Who is reviewing it? 
@TonyMorrison 
@LinkMJB 
@SmithJosh 
@codice/continuous-integration 
@codice/build

---
This change is [Reviewable](https://reviewable.io/reviews/codice/alliance/886)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

**Review by TonyMorrison — APPROVED** [review]
If skipping these modules allows the rest to work then I am cool

File: Jenkinsfile [hunk 1]
```diff
@@ -154,7 +154,7 @@ pipeline {
             }
             steps {
                 withMaven(maven: 'maven-latest', jdk: 'jdk11', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                            sh 'mvn -q -B -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=org.codice:alliance -Dsonar.exclusions=${COVERAGE_EXCLUSIONS} -pl !distribution/alliance,!$DOCS,!$ITESTS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                            sh 'mvn -q -B -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=org.codice:alliance -Dsonar.exclusions=${COVERAGE_EXCLUSIONS} -pl !$DOCS,!$ITESTS,!distribution/alliance,!distribution/test/itests/test-itests-alliance,!distribution/test/itests/test-itests-common,!distribution/test/itests/test-itests-dependencies-app $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```
Comment (SmithJosh) [inline Jenkinsfile:157]: You can skip all the itests with `-P !itests` like the "Dependency Check" stage does, as of https://github.com/codice/alliance/pull/870

Looks like I missed the SonarCloud stage when making those changes. Sorry about that

**Comment by shaundmorris** [comment]
build now

File: Jenkinsfile (hunk 1 above)
Comment (SmithJosh) [reply Jenkinsfile]: Yep, that looks good

**Review by SmithJosh — APPROVED** [review]

**Comment by shaundmorris** [comment]
CI build passed but the status on the PR wasn't updated. Merging.

**Pull request merged**